HopsanGUI/LogVariable.cpp

```diff
@@ -1216,6 +1216,20 @@ bool VectorVariable::positiveNonZeroMinMaxOfData(double &rMin, double &rMax, int
     return ((rMinIdx > -1) && (rMaxIdx>-1));
 }
 
+double VectorVariable::rmsOfData() const
+{
+    QVector<double> *pData = mpCachedDataVector->beginFullVectorOperation();
+    double rms;
+    for (int i=0; i<pData->size(); ++i)
+    {
+        rms += (*pData)[i]*(*pData)[i];
+    }
+    rms /= pData->size();
```

Comment:
not sure if it can happen, but if size = 0 then this will cause unexpected results

Reply:
I added a check which returns zero if data is empty. Also fixed uninitialized variable (`rms`).